host/sgx/create.c

```diff
@@ -401,6 +402,41 @@ static oe_result_t _initialize_enclave(oe_enclave_t* enclave)
     return result;
 }
 
+/*
+** _config_enclave()
+**
+** Config the enclave with an array of configurations.
+*/
+static oe_result_t _configure_enclave(
+    oe_enclave_t* enclave,
+    const oe_enclave_config_t* configs,
+    uint32_t config_count)
+{
+    oe_result_t result = OE_UNEXPECTED;
+
+    for (uint32_t i = 0; i < config_count; i++)
+    {
+        switch (configs[i].config_type)
+        {
+            // Configure the switchless calls, such as the number of workers.
+            case OE_ENCLAVE_CONFIG_CONTEXT_SWITCHLESS:
+            {
+                size_t max_host_workers =
+                    configs[i].u.context_switchless_config->max_enclave_workers;
```

Comment:
Shouldn't this be 
> configs[i].u.context_switchless_config->max_**host**_workers;

Also, until we implement support for using the `max_enclave_workers` member, especially if we are shipping v0.7 without support for it, we should explicitly block `oe_create_enclave()` with non-zero `max_enclave_workers` and document it as such in the data structure doxygen.

Reply:
Ahh, we get away with this typo because we set both max_host_workers and max_enclave_workers to 2. Thanks for catching that.